tests/test_input.py

```diff
@@ -860,7 +950,7 @@ def test_config_parsing_relative_input_paths():
                            'learners': "['LogisticRegression']",
                            'log': output_dir,
                            'results': output_dir,
-                           'objective': 'f1_score_macro'}
+                           'objective': 'f1_score_micro'}
```

Comment:
Because, `fill_in_config_option` calls `_setup_config_parser` and `check_config_parsing_key_error` also calls `_setup_config_parser`. In this case, the config file has `objective=accuracy`. So, in the first call to `_setup_config_parser`, it makes `objectives=['accuracy']`, and in the second pass, it has already filled up `objective=`f1_score_macro`, and this creates conflicts and throws error. So, I made the second objective to default so it passes successfully.